src/game/g_cmds.c

```diff
@@ -4296,14 +4296,8 @@ qboolean G_PushPlayer(gentity_t *ent, gentity_t *victim)
 		return qfalse;
 	}
 
-	// Prevent possible cheating, as well as annoying push after revive and spawning
-	if (ent->client->ps.powerups[PW_INVULNERABLE])
-	{
-		return qfalse;
-	}
-
-	// if a player cannot move at this moment, don't allow him to get pushed..
-	if (victim->client->ps.pm_flags & PMF_TIME_LOCKPLAYER)
```

Comment:
The PMF_TIME_LOCKPLAYER should also be left in the check. So, that if for any reason the player is locked, then he/she is really locked in place.

Reply:
Do you imply any future usage of the flag with this? Because as I already pointed out, it's only ever set on revives (pause for example uses `PM_FREEZE` pm_flags).

Reply:
Isn't able to be set via lua?

Reply:
Right, didn't consider that - will add it back.